webaccess/res/keypad.html

```diff
@@ -95,19 +95,19 @@
   websocket.onopen = function(ev) {
     isConnected = true;
   };
-  websocket.onclose = function(ev) {
-    console.log(
-      "QLC+ connection is closed. Reconnect will be attempted in 1 second.",
-      e.reason
-    );
+
+  websocket.onclose = function (ev) {
```

Comment:
Seems like ev is not used here and in onerror.
Please either use it or remove it, thanks